carto/sql.py

```diff
@@ -226,3 +243,200 @@ def cancel(self, job_id):
         """
         confirmation = self.send(self.api_url + job_id, http_method="DELETE")
         return confirmation['status']
+
+
+class CopySQLClient(object):
+    """
+    Allows to use the PostgreSQL COPY command for efficient streaming
+    of data to and from CARTO.
+    """
+
+    def __init__(self, client, api_version='v2'):
+        """
+        :param client: Auth client to make authorized requests, such as
+                        APIKeyAuthClient
+        :param api_version: Current version is 'v2'. 'v1' can be used to avoid
+                            caching, but it's not guaranteed to work
```

Comment:
No, it is not. But indeed that should be transparent to the users and it does *not* work with v1 (because of how nginx is configured).

I just kept that for regularity with the rest of the API.